src/conf/local-file.yaml

```diff
@@ -37,7 +37,7 @@ observability:
   cron-job-time-interval: "0h0m10s"         # format: XhYmZs
   dbname: ./accuknox-obs.db
   system-observability: true
-  network-observability: true
+  network-observability: false
```

Comment:
do we want to keep network observability off by default?

Reply:
This is kept off temporarily until full validation. Once validation is complete, will be turned ON.

Reply:
Set it to TRUE now